src/test/java/calculatortest/CalculatorTest.java

```diff
@@ -0,0 +1,73 @@
+package calculatortest;
```

Comment:
테스트 패키지명은 프로덕션 코드의 패키지명과 일치 시켜 주시면 좋을 것 같아요
```suggestion
package calculator;
```